packages/components/src/form-token-field/suggestions-list.tsx

```diff
@@ -156,6 +157,11 @@ export function SuggestionsList<
 				);
 				/* eslint-enable jsx-a11y/click-events-have-key-events */
 			} ) }
+			{ suggestions.length === 0 && (
+				<li className="components-form-token-field__suggestion is-empty">
```

Comment:
Were you planning on styling the `is-empty` modifier? I'm thinking we at least need to move the cursor style:

```scss
	&:not(.is-empty) {
		cursor: pointer;
	}
```

Reply:
the class is more of placeholder so that where the component is used the user can easily overide the styling with `is-empty` class.  But your suggestion here make sense, I'll add it

Reply:
> where the component is used the user can easily overide the styling with is-empty class.

That is something that we actively discourage — internal classnames are not public APIs and should not be considered as such